index.bs

```diff
@@ -240,344 +240,160 @@ and across different user settings for default text size.
 <h3 id="simplicity">Prefer simple solutions</h3>
 <!-- was "Avoid Needless Complexity" in the HTML Design Principles -->
 
+Look hard for simple solutions to the
+<a href="#priority-of-constituencies">user needs</a>
+you intend to address.
+
 Simple solutions are generally better than complex solutions,
 although they may be harder to find.
 Simpler features are easier for user agents to implement and test,
 more likely to be interoperable,
-and easier for authors to understand. See related [[LEAST-POWER]]
-
-<h3 id="naming-is-hard">Naming things</h3>
-
-Names take meaning from:
-
-* signposting (the name itself)
-* use (how people come to understand the name over time)
-* context (the object on the left-hand side, for example)
-
-<h4 id="naming-common-words" class="no-num no-toc">Use common words</h4>
+and easier for authors to understand.
 
-API naming *must* be done in easily readable US English.
-Keep in mind that most web developers are not native English speakers.
-Whenever possible, names should be chosen that use common vocabulary
-non-native English speakers are likely to understand when first encountering the name.
+Make sure that your <a href="#priority-of-constituencies">user needs</a>
+are well-defined, so that you can avoid scope creep.
```

Comment:
Not sure what would be the best wording, but assuming scope creep is overengineering, there is also the problem of underengineering. (e.g. not taking non-latin languages into account for an API that deals with strings)